docs/user/security/authorization/index.asciidoc

```diff
@@ -6,22 +6,6 @@ The Elastic Stack comes with the `kibana_admin` {ref}/built-in-roles.html[built-
 
 When you assign a user multiple roles, the user receives a union of the roles’ privileges. Therefore, assigning the `kibana_admin` role in addition to a custom role that grants {kib} privileges is ineffective because `kibana_admin` has access to all the features in all spaces.
 
-[[xpack-security-multiple-tenants]]
-==== Supporting multiple tenants
```

Comment:
**question:** @arisonl do you think we should add a statement to the docs stating that multi-tenancy isn't supported, or is it sufficient to remove this and leave the rest unsaid?